frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -340,18 +412,22 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
   };
 
   const showMonitoringCheckbox =
-    operatorRequestsMonitoring && _.startsWith(selectedTargetNamespace, 'openshift-');
+    operatorRequestsMonitoring &&
+    !suggestedNamespaceRequestsMonitoring(suggestedNamespace) &&
+    _.startsWith(selectedTargetNamespace, 'openshift-');
 
-  const createNamespaceDetails = isSuggestedNamespaceSelected && !suggestedNamespaceExists && (
```

Comment:
Here I was in an impression that we wanna show the monitoring checkbox even if the suggested namespace exists. Thats why Im PATCHing the suggested namespace in that case.

> When this annotation is set to true, the OpenShift Console will update the namespace that the operator is being deployed to with the openshift.io/cluster-monitoring=true label.

Reply:
We intentionally didn't do this before because it enables monitoring for the entire namespace, and we don't know what workloads might already be running.

@shawn-hurley @bparees Any opinion on this?

Reply:
Correct, we need to be extra careful when we give folks the ability to add metrics to the cluster monitoring stack. I think that we should keep the logic that we only allow you to add monitoring if the namespace does not exist.

Reply:
>  I think that we should keep the logic that we only allow you to add monitoring if the namespace does not exist.

on the one hand it sounds reasonable to me, on the other hand we make no restrictions on adding workloads to a namespace that has monitoring enabled, so while we protect you in one direction(keep you from turning monitoring on when existing untrusted workloads may exist), we provide no protection in the other direction(keeping you from adding untrusted workloads to a monitored namespace).  Something to think about for the future, perhaps.

Reply:
cc @lilic @s-urbaniak